starky/src/f3g.rs

```diff
@@ -137,7 +166,22 @@ impl F3G {
     }
 
     #[inline]
-    pub fn batch_inverse(elems: &[Self]) -> Vec<Self> {
+    fn inv(self) -> Self {
+        self._inv()
+    }
+
+    #[inline]
+    fn as_int(&self) -> u64 {
+        self._as_int()
```

Comment:
Can you move the function body of _xxxx(e.g. _as_int) into xxxx, and remove the _xxxx, hence the code would be more simplication